src/attributes/classes.js

```diff
@@ -78,7 +76,7 @@ jQuery.fn.extend({
 
 				if ( cur ) {
 					j = 0;
-					while ( (clazz = classes[j++]) ) {
+					while ( ( clazz = classes[ j++ ] ) ) {
```

Comment:
Understood. I am in favor of continuing to wrap statement-head assignment expressions with parentheses.